src/main/java/duke/tasks/Task.java

```diff
@@ -0,0 +1,44 @@
+package duke.tasks;
+
+public class Task {
+    private String item;
+    private String status;
+    private String type;
+
+    public Task(String item) {
+        this.item = item;
+    }
+
+    public void printTask() {
+        System.out.print("[" + this.getType() + "] "
+                + "[" + this.getStatus() + "] "
+                + this.getItem());
+    }
+
+    @Override
+    public String toString() {
+        int statusAsInt = this.getStatus() == "X" ? 1 : 0;
```

Comment:
Avoid magic numbers. In this case you'd probably want to declare what 1 and 0 mean.